api/task_manager.cc

```diff
@@ -231,7 +232,8 @@ void set_task_manager(http_context& ctx, routes& r, sharded<tasks::task_manager>
         while (!q.empty()) {
             auto& current = q.front();
             res.push_back(co_await retrieve_status(current));
-            for (auto& child: current->get_children()) {
+            for (auto& child_entry: current->get_children()) {
```

Comment:
`for (auto& [_, child] : current->get_children())`

also, style nit: space around `:`